### PR TITLE
fix GetAll() potentially return an unprotected map

### PR DESCRIPTION
**Pull request by gavwu**
#137

### CODE REVIEW

File: pkg/cover/store.go [hunk 1]
```diff
@@ -264,9 +264,13 @@ func (l *memoryStore) Get(name string) []string {
 
 // Get returns all the registered service information
 func (l *memoryStore) GetAll() map[string][]string {
+	res := make(map[string][]string)
 	l.mu.RLock()
 	defer l.mu.RUnlock()
-	return l.servicesMap
+	for k,v := range l.servicesMap {
+		res[k] = append(make([]string,0,len(v)), v...)
```
Comment (CarlJi) [inline pkg/cover/store.go:271]: suggest to just use 
`res[k]=v`

Comment (gavwu) [reply pkg/cover/store.go:271]: `res[k]=v` is not totally safe if the caller modify the slice with index

Comment (CarlJi) [reply pkg/cover/store.go]: v is not a pointer of the actual value of `l.servicesMap`. FYI: https://github.com/golang/go/wiki/Range#gotchas

Comment (lyyyuna) [reply pkg/cover/store.go]: @CarlJi, v is slice, it is sth like pointer

**Pull request merged**